languages/csharp-dotnet-2.2/template/__ALGO__.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Algorithmia.Client" Version="(1.0,)" />
+    <PackageReference Include="Algorithmia.Pipe"/>

```

Comment:
Do we want to wildcard the version entirely, or just the major? I'd like to push for a minor/revision version wildcard.